app/views/crime_applications/_review_overview.html.erb

```diff
@@ -6,17 +6,36 @@
       </span>
     </div>
 
-    <p class="govuk-body-l govuk-!-margin-bottom-2">
+    <% if crime_application.superseded? %>
+      <% id = 'superseded_' %>
+    <% end %>
+
+    <p class="govuk-body-l govuk-!-margin-bottom-2" id="<%= id %>style">
```

Comment:
Perhaps consider applying the class to the outer div and target elements with CSS instead of adding the class to the markup. Just a suggestion.